week-2-practise/level-3/6-map.js

```diff
@@ -4,12 +4,11 @@
 //    - every resulting item run through the `capitalise` function
 
 function tidyUpString(str) {
```

Comment:
Try to branch from master for each new week, it makes the pull request easier. Going to skip to week 4.